src/test/ui/print_type_sizes/nullable.stdout

```diff
@@ -0,0 +1,24 @@
+print-type-size type: `IndirectNonZero<u32>` overall bytes: 40 align: 8
+print-type-size    field .pre bytes: 8
+print-type-size    field .nested bytes: 24
+print-type-size    field .post bytes: 8
+print-type-size type: `MyOption<IndirectNonZero<u32>>` overall bytes: 40 align: 8
+print-type-size    variant Some exact bytes: 40
+print-type-size        field .0 bytes: 40
```

Comment:
* The "exact" is there because the layout code makes a distinction between minimum size for a type (which is used for dynamically sized types) vs exact size. However, I never actually made a test that illustrates this, and furthermore, it does not seem to ever arise in my runs on the `rustc` crate, so maybe that is all dead code for the purposes of this analysis.

 * You're right about the colon placement. I'll think about your suggestion. (My instinct was to instead add colons after "variant" and "field" but your sample looks okay to me...)

 * I can move the "bytes" around to occur after the number, sure.

 * My reasoning behind the backticks is that we only need them for the types (since those have substructure in the form of their instantiated type parameters); the field and variant names never have any substructure, they are always just identifiers or numbers (I think?) so I figured no backticks needed. Note that in your sample you did not put "\`.0\`"

 * A comma before align/alignment sounds fine.

 * Thanks, I thought I fixed the inconsistent indents but clearly never bothered to count spaces in the actual output in the end.